utils/template.go

```diff
@@ -110,14 +111,6 @@ func RenderTemplate(w http.ResponseWriter, context structs.PageContext, data int
 		"formatTime": func(fmt string, t time.Time) string {
 			return t.Format(fmt)
 		},
-		// TODO(CaptainHayashi): this is temporary
-		"stripHTML": func(s string) string {
```

Comment:
I assume you’ve checked this isn’t used anywhere?

Reply:
Ah yes I have broken something, will fix that now 🤦🏼‍♂️, I thought it was re-declaration

Reply:
oh bugger this new one doesnt handle `'` properly
![Screenshot 2021-09-19 at 19 35 41](https://user-images.githubusercontent.com/10200358/133938951-66146c26-5ae8-4560-ac5d-5f62eb453469.png)

Reply:
Will take another look at this tomorrow, unless someone beats me to it :)